src/components/DraggableLineChart/DraggableLineChart.tsx

```diff
@@ -0,0 +1,170 @@
+import _ from 'lodash';
+import React from 'react';
+import PropTypes from 'react-peek/prop-types';
+import {
+	omitProps,
+	Overwrite,
+	StandardProps,
+} from '../../util/component-types';
+import * as d3Selection from 'd3-selection';
+import { lucidClassNames } from '../../util/style-helpers';
+import DraggableLineChartD3 from './DraggableLineChartD3';
+
+const cx = lucidClassNames.bind('&-DraggableLineChart');
+const { arrayOf, number, object, shape, func } = PropTypes;
+
+interface IDraggableLineChartMargin {
+	top: number;
```

Comment:
wow, tabs?

Reply:
Hm weird. Shouldn't this be addressed by `prettier`?

Reply:
yes but It might not have ran on commit

Reply:
Weird. Ran it locally to be sure and it shows no diff with this branch but it looks like no tabs on my end

Reply:
prettier might not be setup on this repo.